### ISSUE TITLE
Add 1R2C and 2R3C options to "class-of-service"?

### ISSUE THREAD

**Issue by alreadytiredofnomcom** [alreadytiredofnomcom/L2SMissues#38]
Bin:
“class-of-services”: should we add 1R2C and 2R3C option?

Qin:
If we need to add IR2C and 2R3C, I propose add policer leaf under "service-level" as follows:
"
identity policer {
description
"Identity of policer type";
}

identity one-rate-two-color {
base policer;
description
"Identity of one-rate, two-color (1R2C)"; }

identity two-rate-three-color {
base policer;
description
"Identity of two-rate, three-color (2R3C)"; } Container service-level{ leaf policer { type identityref { base policer; } description "The policer can be either one-rate, two-color (1R2C) or two-rate, three-color (2R3C)"; } } "

Also I am wondering if we need to change cos-identifier node type from string to identityref since cos-identifier supports EVC or EVC+CVLAN Do we also need to change color-identifier node type from string to identityref sicne color-identifier supports EVC or EVC + CVLAN?

```
     |           |  +--rw service-level
     |           |     +--rw cos-identifier?                  String (do we need to change string into identityref?)
     |           |     +--rw color-identifier?                string
     |           |     +--rw ingress-bw-profile-per-evc?       string
     |           |     +--rw ingress-bw-profile-per-cos-id?     string
     |           |     +--rw egress-bw-profile-per-evc?        string
     |           |     +--rw egress-bw-profile-per-cos-id?      string
     |           |     +--rw byte-offset?                    uint16
     |           |     +--rw COS?                          uint32
```

**Comment by alreadytiredofnomcom** [comment]
Forgot DSCP

Agreed change cos-identifier node type from string to identityref 

Also OVC end point

**Comment by alreadytiredofnomcom** [comment]
Available in 20160926 release
